west.yml

```diff
@@ -56,7 +56,7 @@ manifest:
       revision: f1fa8241f8786198ba41155413243de36ed878a5
       path: modules/hal/infineon
     - name: hal_nordic
-      revision: 4be2a1eadc99511e0271b6de6e0be459a4cf0cef
+      revision: pull/48/head
```

Comment:
please don't forget to update these to new SHAs

Reply:
I'll update it once these pull requests get merged yes, so a tiny bit before this PR gets merged then.